k8s/k8s.go

```diff
@@ -2808,6 +2812,38 @@ func (k *k8sOps) DeleteClusterPair(name string) error {
 	})
 }
 
+func (k *k8sOps) ValidateClusterPair(name string, timeout, retryInterval time.Duration) error {
+	if err := k.initK8sClient(); err != nil {
+		return err
+	}
+	t := func() (interface{}, bool, error) {
+		clusterPair, err := k.GetClusterPair(name)
+		if err != nil {
+			return "", true, err
+		}
+
+		if clusterPair.Status.SchedulerStatus == v1alpha1.ClusterPairStatusReady && clusterPair.Status.StorageStatus == v1alpha1.ClusterPairStatusReady {
```

Comment:
Can you split this into multiple lines, doesn't even fit in the review page otherwise. Same below
```
if cond1 &&
   cond2 {
}